Source/JavaScriptCore/llint/LLIntOfflineAsmConfig.h

```diff
@@ -169,3 +169,9 @@
 #else
 #define OFFLINE_ASM_HAVE_FAST_TLS 0
 #endif
+
+#if USE(BUILTIN_FRAME_ADDRESS)
+#define OFFLINE_ASM_USE_BUILTIN_FRAME_ADDRESS 1
+#else
+#define OFFLINE_ASM_USE_BUILTIN_FRAME_ADDRESS 0
+#endif
```

Comment:
This addition makes LLIntAssembly.h bloat significantly (since LLInt generates each combination).